src/test/scala/nl/knaw/dans/easy/agreement/TemplateResolverSpec.scala

```diff
@@ -147,7 +147,7 @@ class TemplateResolverSpec extends UnitSpec with MockFactory with TableDrivenPro
     val emd = mock[MockEasyMetadata]
     val date = mock[EmdDate]
     val rights = mock[EmdRights]
-    emd.getPreferredTitle _ expects() returning "about testing"
+    emd.getPreferredTitle _ expects() returning "about <a href='javascript:alert('XSS')'>XSS</a> escape <b>bold</b> <a href='http.dans.knaw.nl'>linked</a> content"
```

Comment:
You changed the input, but shouldn't this also require a change in the expected output?

Reply:
The tests have other purposes. HTML output is only produced as side effect for (css) debugging purposes.